### DISCUSSION TITLE
New functionality to export to SQL option - append new data to existing sql table

### DISCUSSION THREAD

**Discussion by scbleecker** [DaxStudio/DaxStudio#516] [Ideas]
I have been experimenting with the export data to SQL table option but it seems like it just overwrites the existing data in the sql table instead of adding new records.  Would be great if the option was added to be able to append new data to the existing table.

**Comment by dgosbell** [comment]
This option should already exist. If you untick the option to truncate the tables when starting the export wizard it should just append on to the existing table.

![image](https://user-images.githubusercontent.com/8241830/105414127-312a5700-5c8b-11eb-9c48-83cb3c02e279.png)

**Reply by scbleecker** [reply]
Hi Darren,
Thanks for the reply.  A coworker and myself actually experimented with unchecking that option today and got the same result; just overwrites the existing data in the sql table.  I am new to using SQL server so I could just not be using the tool properly.  That being said I had previously posted my question as a comment to a recent related DAX Studio youtube video from Alberto from SQLBI.com and he was the one that told me the functionality does not currently exist and to post the suggestion to this forum as a future dev possibility.  Link to Alberto's comment attached for reference here.  
https://www.youtube.com/watch?v=WaUCYUHLHdw&lc=UgwtLKUvoMjJHeMppQt4AaABAg.9Il8dji0uv29ImOqyX2Usn&feature=em-comments

**Reply by dgosbell** [reply]
In this case Alberto is mistaken, the functionality does exist - it just has a bug in it. I've created a bug #520 to track this, I can see where the issue is and it should not be too hard to fix.

**Reply by scbleecker** [reply]
That would be great!
Any rough idea how long it might take to fix the bug? I’m working on a solution for a project at work and was about to move on to trying to use sql server management studio instead, but I have no idea how to use that program.

**Reply by dgosbell** [reply]
It's already fixed in the source code, I'm not exactly sure when I will push out the next update as I have a couple of other issues I'm investigating. But there is a preview build up on our build server if you would like to try that.

You can download it from the link below, it's basically the same as 2.14.1 plus this one fix:
https://ci.appveyor.com/project/DarrenGosbell/daxstudio/builds/37389519/artifacts

**Reply by scbleecker** [reply]
I'll check out the preview build.  Thanks Darren!

**Reply by scbleecker** [reply]
FYI...Downloaded the preview build and tested this am.  Seems to be working as expected now!  Thanks again!
